src/addresses.py

```diff
@@ -0,0 +1,175 @@
+import sys
+
+import pyarrow
+from osmium.osm import OSMObject, Relation
+from .geoparquet import GeoParquetWriter
+
+
+class AddressesWriter(GeoParquetWriter):
+    COLUMNS = [
+        ("addr:housenumber", pyarrow.string()),
```

Comment:
When a feature has many house numbers, some mappers and imports enumerate all of them, while others express a range. Is there any interest in either parsing value lists and ranges into something more structured or consolidating them into a human-readable value? Same for `addr:unit=*`.

/ref openmaptiles/openmaptiles#1562